Packages/com.unity.inputsystem/InputSystem/Plugins/XR/TrackedPoseDriver.cs

```diff
@@ -102,6 +103,7 @@ void BindRotation()
         {
             if (!m_RotationBound && m_RotationAction != null)
             {
+                m_RotationAction.Rename($"{gameObject.name} - TPD - Rotation");
```

Comment:
Alternatively, do the renaming on the serialized data? e.g. in `TrackedPoseDriverEditor` or `OnValidate`.